labman/gui/static/js/plateList.js

```diff
@@ -0,0 +1,44 @@
+var gPlateInfoByPlateId = {};
```

Comment:
Does this variable need to be cleared somewhere else in the UI or is an initialization per-page-load sufficient to guarantee correct functionality.

Reply:
This is a good question.  I believe that initialization per-page-load is sufficient, particularly because this variable is being accessed by specific keys; there is no time when the code loops over ALL keys in the dictionary without regard for whether they are also in the explicit list of plates to display, so even if there were some records left over for e.g. removed plates, they should not be accessed or acted upon.  

Now, that said, it is always possible someone down the line could change how this variable is being accessed.  I'm open to suggestions if there's a variable-management approach you think would be preferable!

Reply:
I think this should be fine then. In order to get rid of this global variable I think we would need to refactor the way the add plate functionality is working so that everything was in the same scope. Perhaps my only suggestion would be to add a comment in line 71 of `labman/gui/templates/compression.html` noting where this variable is being declared/populated.

Reply:
Sold :)